data/wanderer/wanderers start.txt

```diff
@@ -2926,3 +2930,13 @@ mission "Wanderers Ap'arak 3"
 		event "wanderers: spera anatrusk colony" 40
 		conversation
 			`A few minutes after you land on <planet>, a sudden uproar passes through the Wanderers in the spaceport. It's hard to be sure, but you think they sound excited, not frightened. Ships start taking off, more and more of them until the spaceport is almost deserted, and no one who is left seems at all interested in talking with you or explaining what is going on. You're probably going to have to return to your ship and see for yourself what all the fuss is about.`
+
+# Withdraw Wanderer defense fleets from Unfettered systems on pre-0.10.4 pilots.
+mission "Wanderers Retreat Patch"
+	landing
+	invisible
+	to offer
+		has "Wanderers Ap'arak 2: done"
```

Comment:
Won't this mission also trigger on new save files as well?

Reply:
Yes, but there's no harm in calling `remove` on a fleet twice. I also can't think of how we'd cause this to only offer on old save files.